pkg/agent/openflow/fields.go

```diff
@@ -139,17 +144,17 @@ var (
 
 // Marks using CT.
 var (
-	//TODO: There is a bug in libOpenflow when CT_MARK range is from 0 to 0, and a wrong mask will be got,
-	// so bit 0 of CT_MARK is not used for now.
+	//TODO: There is a bug in libOpenflow when CT_MARK range is from 0 to 0, and a wrong mask will be got. As a result,
+	// don't just use bit 0 of CT_MARK.
 
 	// Mark to indicate the connection is initiated through the host gateway interface
 	// (i.e. for which the first packet of the connection was received through the gateway).
-	FromGatewayCTMark = binding.NewCTMark(0b1, 1, 1)
-	// Mark to indicate DNAT is performed on the connection for Service.
-	ServiceCTMark = binding.NewCTMark(0b1, 2, 2)
+	FromGatewayCTMark = binding.NewCTMark(fromGatewayVal, 0, 3)
```

Comment:
Could we comment we allocate same bits and positions as `PktSourceField` for ease of persisting the value from reg0 to CTMark in case they are updated by accident in future?

Reply:
Updpated.